src/main.cpp

```diff
@@ -1,9 +1,18 @@
 #include <Arduino.h>
+#include "Led.h"
 
-void setup() {
-  // put your setup code here, to run once:
+// Define output pin and period of blink
+#define Pinout 1
+#define led_delay 50 // This value x2 is the period in ms
+
+// Create Led instance
+Led led(Pinout);
```

Comment:
is it possible to move this to setup()?

Reply:
No this is not possible. This line creates an instance of the Led class, which you cannot do within the setup().